### PR TITLE
Find library and includes with `pkg-config`; fix build without `custom_logger`

### PR DESCRIPTION
**Pull request by ids1024**

### CODE REVIEW

File: libseat-sys/build.rs [hunk 1]
```diff
@@ -25,4 +32,5 @@ fn main() {
 #[cfg(not(feature = "use_bindgen"))]
 fn main() {
     println!("cargo:rustc-link-lib=seat");
```
Comment (PolyMeilex) [inline libseat-sys/build.rs]: Is this still needed? As far as I can tell, pkg_config will link the lib in `probe_library() -> Config::probe() -> Library::parse_libs_cflags()`.
Same goes for bindgen build

Feel free to correct me if I missed the purpose somehow.

**Review by PolyMeilex — COMMENTED** [review]
Thanks, this looks great!
Just a single comment/question, and it should be good to go.

File: libseat-sys/build.rs (hunk 1 above)
Comment (ids1024) [reply libseat-sys/build.rs]: Right, it seems the `pkg_config` adds this automatically.

Updated.

**Comment by PolyMeilex** [comment]
Thanks, will do patch release shortly

**Pull request merged**